graceful-grasshoppers/README.md

```diff
@@ -1,2 +1,114 @@
-# Graceful Grasshoppers
-This is the default README of your team's project. Please replace this by a README with more information on your project. At the very least, your README should contain information on how to set-up and run your project.
+
+# PySpace!
+
+## Info
+Our project is a social media platform, based off of the popular MySpace website, hence the name of our project; PySpace. On our platform users can create posts, like each others posts, make comments on each others posts, and follow each other, just as they could on our inspiration site, myspace. 
+
+## Setup:
+In order to setup this project you'd need to have nodejs installed (and python of course).
+Node.js Installation Link: https://nodejs.org/en/download/.
+
+first we'll start with running the backend (or in other words, the API):
+1. Open your preffered console/terminal.
+2. cd into the PySpace directory - Example if you've installed the summer code jam fork on your Desktop:
+```
+cd path/to/desktop/summer-code-jam-2020/graceful-grasshoppers/PySpace
+```
+3. Create a virtual environment.
+- MacOS or Linux (bash): 
+```
+python3 -m venv pyspace
+source venv/bin/activate
+```
+- Windows (cmd): 
+```
+py -m venv pyspace
+.\venv\Scripts\activate
+```
+Now you have your virtual environment activated. (to be sure, make sure the prompt is now `(pyspace) the/path/of/the/current/folder`)
+4. Install the requirements.
+```
+pip install -r requirements.txt
```

Comment:
Alternatively, consider using a dependency management tool such as `pipenv`, which would also allow for custom run scripts, which would simplify some installation steps.